docs(faq): add invoice status to billing faq (#1065)
scaleway/docs-content: faq/billing.mdx

Our reference billing period is the definition of a calendar month in the Coordinated Universal Time (UTC) time zone.
The automatic payment of invoices occurs in the first few days of a month, once the invoice is issued and is processed by using the preferred payment method of your account.

## What does the status of my invoice mean?

The status of an invoice can be:

> * **Draft**: Concerns the invoice for the current calendar month. The billing period has not yet finished, so the invoice is not finalized and is subject to change.

Reviewer: ```suggestion
* **Draft**: regards the invoice for the current calendar month. The billing period has not yet finished, so the invoice is not finalized and is subject to change.
```

Reviewer 2: I'm not sure about this suggestion, I think concerns makes more sense than regards?

Author: +1 with @RoRoJ

> * **Stopped**: Concerns invoices where the billing period (calendar month) has ended, but the invoice is not yet issued.

Reviewer: ```suggestion
* **Stopped**: regards invoices where the billing period (calendar month) has ended, but the invoice is not yet issued.
```

Author: ```suggestion
* **Stopped**: Refers to invoices where the billing period (calendar month) has ended, but the invoice is not yet issued.
```

> * **Issued**: Concerns a final invoice awaiting payment. It has a unique number, and is non-repudiable. The invoice charges for the resources consumed over the corresponding billing period.

Reviewer: ```suggestion
* **Issued**: regards a final invoice awaiting payment. It has a unique number, and is non-repudiable. The invoice charges for the resources consumed over the corresponding billing period.
```

Author: ```suggestion
* **Issued**: Refers to a final invoice awaiting payment. It has a unique number, and is non-repudiable. The invoice charges for the resources consumed over the corresponding billing period.
```

> * **Paid**: Concerns a fully paid invoice. No further action is required.

Reviewer: ```suggestion
* **Paid**: regards a fully paid invoice. No further action is required.
```

Author: ```suggestion
* **Paid**: Refers to a fully paid invoice. No further action is required.
```

> * **Incomplete**: Concerns invoices where the user has provided incorrect billing details or payment information, or elements of this information are missing. The user must update this information as soon as possible to avoid having their account deleted.

Reviewer: ```suggestion
* **Incomplete**: regards invoices where the user has provided incorrect billing details or payment information, or elements of this information are missing. The user must update this information as soon as possible to avoid having their account deleted.
```

Author: ```suggestion
* **Incomplete**: Refers to invoices where the user has provided incorrect billing details or payment information, or elements of this information are missing. The user must update this information as soon as possible to avoid having their account deleted.
```

> * **Outdated**: Concerns invoices where there has been a change in billing information, for example a discount has been applied, or the client has amended their billing details. An updated invoice will be issued soon.

Reviewer: ```suggestion
* **Outdated**: regards invoices where there has been a change in billing information, for example a discount has been applied, or the client has amended their billing details. An updated invoice will be issued soon.
```

Author: ```suggestion
* **Outdated**: Refers to invoices where there has been a change in billing information, for example a discount has been applied, or the client has amended their billing details. An updated invoice will be issued soon.
```
